cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -542,13 +640,17 @@ subroutine dyn_prep2 (nx_block,   ny_block,   &
       !-----------------------------------------------------------------
 
       icellu = 0
+         
       do j = jlo, jhi
       do i = ilo, ihi
-
-         ! ice extent mask (U-cells)
          iceumask_old(i,j) = iceumask(i,j) ! save
+!         if (grid_ice == 'B') then ! include ice mask.
```

Comment:
It looks like this set of changes should be reverted or at least cleaned up.

Reply:
I noticed this too, but wasn't sure if it's useful to keep or not.  I'm happy to remove it as long as it's not acting like a TODO.